### DISCUSSION TITLE
Questions on Input files

### DISCUSSION THREAD

**Discussion by XuanhongZhou** [UMEP-dev/UMEP#691] [Q&A]
Dear developer, I am using the SUEWS_BiogenCO2 part of the SUEWS model for my Biogenic carbon flux model.
The code has the following judgement statement, I would like to ask if I can set the EmissionMethod to be the same value for each site if my study area is fixed to a particular city, so that I can save some preparation for inputting variables：
```
if 11 <= EmissionsMethod <= 16:
elif 21 <= EmissionsMethod <= 26:
elif 31 <= EmissionsMethod <= 36:
elif 41 <= EmissionsMethod <= 46:
```
Also, I noticed that the description gives the reference values 0, 1, 2, 3, 4, 45 for the EmissionMethod, which doesn't seem to fit well with the judgement branch in the code
![image](https://github.com/user-attachments/assets/62d53a4d-6aef-406b-9199-7d2046a2ae81)
Very much looking forward to the author's reply, thank you very much!
@sunt05

**Comment by XuanhongZhou** [comment]
@biglimp 
Very much looking forward to your reply, thank you very much!

**Comment by sunt05** [comment]
Thank you for raising this question. Since the SUEWS code is open-source, you are free to adapt it to meet your specific needs. Here are some suggestions:

- To assign a uniform `EmissionsMethod` across all sites, you can modify the code accordingly.
- For discrepancies between the `EmissionsMethod` values in the documentation and the code, review the conditional statements and adjust them to suit your usage.

While we can’t provide direct support for custom development, we encourage you to experiment and share your insights with the community. Collaborative discussions can benefit everyone.

Good luck with your work!

**Comment by XuanhongZhou** [comment]
Dear developer，the code in SUEWS_BiogenCO2 part of the SUEWS model: 
![image](https://github.com/user-attachments/assets/fa32a78a-1766-46be-818f-92b194a459f6)
We only have data on the proportions of various vegetation types, not bsoilsurf. Is it possible to ignore the effect of bsoilsurf when calculating vegetation photosynthesis？
Very much looking forward to your reply, thank you very much!
@sunt05 @biglimp

**Reply by sunt05** [reply]
The code was designed for the SUEWS structure, so if your application differs, feel free to modify it as needed. However, ensure that changes align with the underlying physics, particularly how bsoil influences photosynthesis. Review Belluco et al. (2017) for context before making adjustments.
